scripts/solc-bump.js

```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const readline = require('readline');
+
+const PACKAGES_DIR = `${__dirname}/../packages`;
```

Comment:
Should use `path.resolve()` for this to make it more cross-platform.

Reply:
Good call, updated